app.rb

```diff
@@ -0,0 +1,166 @@
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'classroom'
+require_relative 'person'
+
+class App
+  attr_accessor :books, :rentals, :people
+
+  def initialize
+    @books = []
+    @rentals = []
+    @people = []
+  end
+
+  def select_section(section)
+    case section
+    when '1'
+      books_list
+    when '2'
+      people_list
+    when '3'
+      create_person
+    when '4'
+      create_book
+    when '5'
+      create_rental
+    when '6'
+      rentals_list
+    else
+      puts 'Please select a valid section!'
+    end
+  end
+
+  def books_list
+    @books.each do |book|
+      book.instance_variables.each do |var|
+        string = ''
+        value = book.instance_variable_get(var)
+        var = var.to_s.delete('@')
+        var = var.capitalize
+        string += "#{var}:#{value} " unless var.include?('Rentals')
+        puts string
+      end
+    end
+  end
+
+  def people_list
+    @people.each do |person|
+      person.instance_variables.each do |var|
+        string = ''
+        value = person.instance_variable_get(var)
+        var = var.to_s.delete('@')
+        var = var.capitalize
+        string += "#{var}:#{value} " unless var.include?('Rentals') or var.include?('Classroom')
+        puts string
+      end
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student(1) or a teacher(2)?'
+    choise = gets.chomp
+
+    case choise
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Your input is wrong!'
+    end
+  end
+
+  def create_student
+    puts 'Age:'
+    age = gets.chomp
+    puts 'Name:'
+    name = gets.chomp
+    puts 'Has parent permission? [Y/N]'
+    parent_permission = gets.chomp.upcase == 'Y'
+    student = Student.new(age, 'Unknown', parent_permission, name)
+    @people.push(student)
```

Comment:
- [ ] There is a problem now while creating a new student the name is saved as unknown, So please fix that.
[Hint] You can edit line 83 to be:
```ruby
 student = Student.new(age, name, parent_permission)
```